wagtail/admin/templates/wagtailadmin/shared/keyboard_shortcuts_dialog.html

```diff
@@ -0,0 +1,28 @@
+{% load wagtailadmin_tags i18n %}
+{% dialog icon_name='regex' id='keyboard-shortcuts-dialog' title=_("Keyboard shortcuts") %}
```

Comment:
I suggest we put the root `classname="w-keyboard-shortcuts"`  on the dialog here, instead of on the `table`. Will make it easier to style the content as a whole.